cmd/bosun/sched/notify.go

```diff
@@ -102,12 +103,46 @@ func (s *Schedule) sendNotifications(rh *RunHistory, silenced map[expr.AlertKey]
 				s.AddNotification(ak, n, time.Now().UTC())
 			}
 		}
+		var c int64
+		tHit := false
+		oTSets := make(map[string]expr.AlertKeys)
 		for name, group := range ustates.GroupSets() {
-			s.unotify(name, group, n)
+			c++
+			if !tHit && (int64(len(ustates.GroupSets()))-c == 1) {
```

Comment:
Does it make sense to just count at the beginning? If len(ustates.GroupSets()) > conf.UnknownThreshold, then just send the one email? Maybe.